sqeleton/databases/postgresql.py

```diff
@@ -122,3 +123,27 @@ def create_connection(self):
             return c
         except pg.OperationalError as e:
             raise ConnectError(*e.args) from e
+
+    def select_table_schema(self, path: DbPath) -> str:
+        database, schema, table = self._normalize_table_path(path)
+
+        info_schema_path = ["information_schema", "columns"]
+        if database:
+            info_schema_path.insert(0, database)
+
+        return (
+            f"SELECT column_name, data_type, datetime_precision, numeric_precision, numeric_scale FROM {'.'.join(info_schema_path)} "
```

Comment:
Just noticed you are using `.lower()` on the names. But I don't think it's right for postgresql.

And reason you added these, or is it just due to copy-paste?

Reply:
I saw that they were all lowercase in my local postgres db.

But I see now that quoting will make them cased:
![Screenshot 2023-02-23 at 11 19 31 AM](https://user-images.githubusercontent.com/11282254/220996251-e3b1e359-e647-402b-b595-287e7fae2499.png)

Reply:
fixed